nginx/Dockerfile.template

```diff
@@ -7,10 +7,11 @@ RUN cp /tmp/config-files/hs-nginx.conf /etc/nginx/conf.d/default.conf \
     && cp /tmp/.htpasswd /etc/nginx/.htpasswd \
     && cp /tmp/docker-entrypoint.sh /docker-entrypoint.sh \
     && groupadd -g SENDFILE_IRODS_GROUP_ID SENDFILE_IRODS_GROUP \
-    && useradd --system --shell /bin/bash -u SENDFILE_IRODS_USER_ID -g SENDFILE_IRODS_GROUP_ID SENDFILE_IRODS_USER 
+    && echo 'SENDFILE_IRODS_USER:x:SENDFILE_IRODS_USER_ID:SENDFILE_IRODS_GROUP_ID::/home/SENDFILE_IRODS_USER:/bin/bash' >> /etc/passwd
```

Comment:
this seems messy but I suppose as long as it works

Reply:
useradd won't work with UID > 65535 (on CUAHSI dev-hs-* most of domain UID is greater than 65535) :( so modify /etc/passwd directly is the solution